source/features/add-switch-branches-on-compare.js

```diff
@@ -0,0 +1,20 @@
+import {h} from 'dom-chef';
+import select from 'select-dom';
+
+import * as pageDetect from '../libs/page-detect';
+
+export default async () => {
+	const compareIcon = select('.octicon-git-compare');
+
+	const compareURL = pageDetect.getRepoPath().replace('compare/', '').split('...');
+
+	// Compares against the "base" branch only have one in the URL
+	const currentBase = compareURL.length > 1 ? compareURL[0] : select('.branch span').textContent;
+	const currentCompare = compareURL[compareURL.length - 1];
+
+	const link = <a href={`/${pageDetect.getRepoURL()}/compare/${currentCompare}...${currentBase}`}></a>;
+
+	// Wrap the icon in our new link
+	compareIcon.parentNode.insertBefore(link, compareIcon);
+	link.appendChild(compareIcon);
```

Comment:
There's a dedicated `wrap` function in `utils.js`

Reply:
Not sure how I missed that, figured we'd have one.